index.html

```diff
@@ -0,0 +1,214 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Birds?</title>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Karla:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./normalize.css">
+    <link rel="stylesheet" href="styles.css">
+  </head>
+  <main>
+    <body>
+      <header>
+        <section class="nav-block">
```

Comment:
For this nav block, maybe change it to the semantic html of `<nav>`. It can either be nested inside of` <header>` or outside. It's all about preference. This change will also be better for accessibility and screen readers.